pkgs/top-level/all-packages.nix

```diff
@@ -1039,6 +1039,10 @@ in
 
   analog = callPackage ../tools/admin/analog {};
 
+  android-tools = lowPrio (callPackage ../tools/misc/android-tools {
+    stdenv = if stdenv.targetPlatform.isAarch64 then gcc10Stdenv else stdenv;
```

Comment:
Why is this needed?

Reply:
Was likely required for https://github.com/NixOS/nixpkgs/pull/124992#issuecomment-852043128 ;)